### PR TITLE
Updated the SwiftUI docs for sprint 15

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🔗 Issue Link
https://stream-io.atlassian.net/jira/software/projects/SWUI/boards/109

### 🎯 Goal

Update the docs based on the latest SwiftUI SDK changes.

### 🛠 Implementation

Updated the docusaurus files.

### 🎨 UI Changes

N/a

### 🧪 Testing

Run the docusaurus docs locally to see the latest changes.

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] This change follows zero ⚠️ policy (required)
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎉 GIF

Alt Text

### CODE REVIEW

**Review by bielikb — APPROVED** [review]

File: docusaurus/docs/iOS/swiftui/components/message-list.md [hunk 1]
```diff
@@ -8,9 +8,43 @@ The message list view in the SwiftUI SDK allows several customization options. T
 
 If you are developing an app with this use-case, you can customize the [message avatars](../custom-avatar), [reactions](../message-reactions), [theming and presentation logic](../../getting-started) and the different types of [attachments](../attachments).
 
+## Directly Showing Channel View
+
+In some cases, you want to show directly the channel view, without having a channel list as the previous screen. Here's an example how to show the channel view as an initial screen, with a predefined channel:
```
[the remark this answers is not in the document]
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/components/message-list.md:13]: Yes, I see your point. When I think about it, this page has all the things around the message list, and maybe adding the code for channel view can be confusing for customers. So the proper solution here would be to move this chunk in separate page. 
Btw, we follow the same convention as UIKit, only the suffix is different (VC vs View, e.g. `ChatChannelVC` vs `ChatChannelView`).

**Review by nuno-vieira — APPROVED** [review]

File: docusaurus/docs/iOS/swiftui/components/message-list.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nuno-vieira) [reply docusaurus/docs/iOS/swiftui/components/message-list.md:13]: Yes I know that, what I mean is that we have 3 doc pages, one for Channel, one for Thread, and one for the Message List, where in SwiftUI there's one page for all

[the remark this answers is not in the document]
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/components/message-list.md:13]: We actually have two at the moment, one for Message List and one for Threads. But I'm soon pushing the new page for Channel too 👍

[the remark this answers is not in the document]
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/components/message-list.md]: It's split up now @nuno-vieira in 3 parts, like in UIKit.

[the remark this answers is not in the document]
Comment (nuno-vieira) [reply docusaurus/docs/iOS/swiftui/components/message-list.md]: Perfect 👌

**Pull request merged**